README.reproducible.md

```diff
@@ -0,0 +1,148 @@
+#Armory Reproducible Build Documentation
+
+##Gitian
+
+Gitian is the build system that Bitcoin Core uses for reproducible builds.
+Armory has adopted it for OS X and Windows reproducible builds. This guide
```

Comment:
Ahhh, I remember now. I suppose that, in that case, the best thing to do would be to have a short introductory paragraph explaining what's going on, and how Gitian is just one method for achieving reproducible builds. The introduction should also explain what we support.